test/utils.h

```diff
@@ -65,19 +70,22 @@ inline TensorView* makeSymbolicTensor(
       s = -1;
     }
   }
+  maybeEmptyAllocator();
```

Comment:
Should we move this to the `TearDown()` of https://github.com/NVIDIA/Fuser/blob/dd25df0c2127fc5adfd86352b29ccaa4043efb9b/test/utils.h#L387

Reply:
Moved to SetUp.